pages/dynamic-portal.tsx

```diff
@@ -532,18 +501,138 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     req: context.req,
   });
 
-  if (!token?.sub) {
+  if (!token) {
     console.log("No session token found");
 
     return {
       notFound: true,
     };
   }
 
-  const accessToken = await getAccessToken();
-  const baseConfig = await getSpaceConfig(accessToken);
-
   const prisma = prismaClient;
+  const environmentToken = process.env.DYNAMIC_TEMPLATES_ENVIRONMENT_ID;
+  const userId = token.sub as string;
```

Comment:
you had to add `as string` here because you changed the if check above

Reply:
reverted